addons/mail/static/tests/discuss/core/public_web/sub_channels.test.js

```diff
@@ -245,3 +245,27 @@ test("muted channel hides sub-thread unless channel is selected or thread has un
     );
     await contains(".o-mail-DiscussSidebar-item:contains('New Thread')");
 });
+
+test("Message author should be thread member when create from message", async () => {
```

Comment:
This test is mostly covering python code which is mocked. I'd rather testing the real code (i.e. calling the `sub-channel/create` route and inspecting the members of the created channel).